Ghidra/Framework/Docking/src/main/java/docking/widgets/fieldpanel/FieldPanel.java

```diff
@@ -1428,7 +1428,16 @@ public void mouseWheelMoved(MouseWheelEvent e) {
 			}
 			else {
 				hoverHandler.stopHover();
-				scrollView(scrollAmount);
+
+				if (e.isShiftDown()) {
+					// horizontal scroll (only move viewport)
+                    if (viewport != null) {
```

Comment:
What's up with the indentation, here?

Reply:
The official [Ghidra Eclipse Formatter](https://github.com/NationalSecurityAgency/ghidra/blob/master/eclipse/GhidraEclipseFormatter.xml).

Reply:
Ok… weird that it uses spaces for one line, and tabs for the others :/

Reply:
Fixed indentation using the formatter. I'm not sure why this happens..